app/Models/Payment.php

```diff
@@ -53,10 +53,9 @@ public function user()
     {
         return $this->belongsTo(User::class);
     }
-
-    public function formatCurrency($locale = 'en_US')
+    public function formatToCurrency($value,$locale = 'de_DE')
```

Comment:
Please change the fallback back to en_US

Reply:
we're working with €-Currency as default. 
You write the €-sign behind the money-amount -> 10,00€ and not €10,00

I  dont see why it should default to US, basically the wrong way around

Reply:
Perhaps we can implement some currency formatting for all countries? Not sure how hard it is to implement but for the proper formatting you can look at Shopify: https://polaris.shopify.com/foundations/formatting-localized-currency

Reply:
its not needed i guess, its not bad having it aswell though.

 it just was there and i did not think about it tbh, just reused it

Reply:
Default 'de_DE'?

This method was to format currency for the payment price. You're no longer using 'this' object but instead made this a global function that no longer belongs in this model since it could be used by anything now.

Usually, I would ask you to move it to some helper class instead, but looking at the code, I think it's okay ;) 

+ missing docblocks